### ISSUE TITLE
Transaction issue

### ISSUE THREAD

**Issue by xxxROACHxxx** [MetaMask/metamask-plugin#1636]
I've tried sending 3 transactions and none go through. This is the latest
transaction. Is there something wrong?Transaction
 0x88f084b35de7999342c310c39186851e203222adf9ca57ddc754b01c7aa07311

**Comment by Zanibas** [comment]
The network is heavily being hit by now by thousands of transactions, so your transaction may not go through right away. The network is trying as fast as it can. Unfortunately, all you can do in this situation is wait. Ask us again tomorrow if the issue still persists!

**Comment by winger13** [comment]
mines the same mate have you had any joy today?

**Comment by xxxROACHxxx** [comment]
No. all my transactions are still in limbo.

**Comment by tmashuang** [comment]
https://github.com/MetaMask/faq/blob/master/USERS.md#slow-transactions

**Comment by xxxROACHxxx** [comment]
In plain English please... what do you propose that I do?

**Comment by Zanibas** [comment]
You could either:
1) Wait for your transaction to be processed by the network. There are currently 6700 transactions waiting to be processed on the network, each competing to get in. We realize it's been several hours, but this is not our fault--it's a fault of the blockchain not being able to process enough transactions quickly enough.

2) Attempt to resubmit your transaction by resetting the state of MetaMask as described in the link above, but submit them with a significantly higher gas price if you are willing to pay more--such as 70GWEI and above. There is no guarantee that this new transaction will be processed before the first one (and thus negating the previous one's validity) or even faster, but until more elegant solutions are proposed, this is what we have.

We are real sorry that people aren't able to get their transactions processed, but much of this is beyond our control and we are as frustrated as you are. =(

**Comment by xxxROACHxxx** [comment]
I understand. Fault doesn't lay with you. My confusion lies with the fact
that I resubmitted the transaction to the same address with higher gwei ...
100 to be exact. Never had to do this before and I'm not sure of the
consequences.

**Comment by Zanibas** [comment]
Two things could happen:
1) The lower gwei tx gets processed first. Your second tx gets cancelled (if they had the same 'nonce' or transaction number). 
2) The higher gwei tx gets processed first and the opposite happens--the first tx is cancelled.

If you didn't resubmit properly and attempted to send the second as a completely different tx distinct from the first, it may be that both end up being processed. If that sounds confusing, we're aware and we're attempting to create UI that better handles cases like this in the future so that it's less confusing to attempt to "redo" transactions.

**Comment by xxxROACHxxx** [comment]
Thank you for the quick reply. I will monitor through out the day.

**Comment by xxxROACHxxx** [comment]
Ok, No transactions are going through. it's been days... I just tried to
send out of metamask to  my ether wallet... still nothing.

**Comment by tmashuang** [comment]
Try switching your network to another and back again, using the network provider menu in the top left of metamask.

**Comment by flyswatter** [comment]
Thomas is correct, and then also retry after.

**Comment by xxxROACHxxx** [comment]
Ty that worked
